sim/paladin/item_sets_pve.go

```diff
@@ -101,3 +101,21 @@ var ItemSetSoulforgeArmor = core.NewItemSet(core.ItemSet{
 		},
 	},
 })
+
+var ItemSetLawbringerRadiance = core.NewItemSet(core.ItemSet{
+	Name: "Lawbringer Radiance",
+	Bonuses: map[int32]core.ApplyEffect{
+		2: func(agent core.Agent) {
+			// No need to model
+			//(2) Set : Your Judgement of Light and Judgement of Wisdom also grant the effects of Judgement of the Crusader.
```

Comment:
Sorry I'm confused why is there nothing to do for this?

Reply:
I suppose it can be modeled. I just don't often see any APL's that explicitly cast these in a rotation and just use the raid debuff settings menu.